snap/snapcraft.yaml

```diff
@@ -0,0 +1,60 @@
+name: matrix-appservice-slack
+base: core18
+version: git 
```

Comment:
What does this mean?

Reply:
`version:  git` tells snapcraft to use the current git commit as the basis for the snap version details, when building a snap. This version then becomes available for pushing to the store and releasing via one of the channels, beta, edge or stable.

The git versioning support in snapcraft detects tags and branches sharing the current commit ID and is able to build regular semvers using that data.

Essentially, this just means that nobody needs to manually set the version on each release, so long as tags are in place which `git describe` will produce a sensible version string for.